README.md

```diff
@@ -1 +1,13 @@
-# sdcore-webui-operator
\ No newline at end of file
+# SD-CORE Webui Operator
```

Comment:
How about making the readmes like the one in the NRF operator? https://github.com/canonical/sdcore-nrf-operator

Reply:
I'm not necessarily against badges but right now they don't really "work".

The problems I see:
- The release in the Github project is marked as 7 but the one in Charmhub is 11 (due to the fact we don't use the charming actions workflow)
- The `publish-charm` is marked as "no-status"

![image](https://github.com/canonical/sdcore-webui-operator/assets/18486508/ff5392c4-dec9-421a-8604-5407f4190db5)

Reply:
Because there are [no publish runs yet](https://github.com/canonical/sdcore-nrf-operator/actions/workflows/publish-charm.yaml), whenever we merge, it should show up as the other one.